pkgs/applications/networking/p2p/magnetico/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, buildGoModule, go-bindata }:
+
+buildGoModule rec {
+  name = "magnetico-${version}";
```

Comment:
```suggestion
  pname = "magnetico";
```

Reply:
Thank you. We should update the [manual](https://nixos.org/nixpkgs/manual/#sec-language-go), then.